Lib/test/test_embed.py

```diff
@@ -1149,7 +1147,6 @@ def test_init_setpath(self):
              # The current getpath.c doesn't determine the stdlib dir
              # in this case.
             'stdlib_dir': '',
-            'use_frozen_modules': -1,
```

Comment:
`Py_SetPath()` should cause this to clear "stdlib_dir", which means "use_frozen_modules" would be -1.  So what changed?

Reply:
This test calls Py_SetPath but then Py_Initialize, which determines that it's not in a build environment and so defaults to using frozen modules.

At one point it was actually picking up the running Python interpreter, which would mean it'd be 1 when testing a regular install and 0 when testing in the build directory. I think it's more stable now, so I can make it check for 1 unconditionally.